.github/workflows/demo.yaml

```diff
@@ -0,0 +1,39 @@
+name: Demo
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - deploy-demo
```

Comment:
@tholor What will be the demo redeploy policy? On every push to master? At every tagged release?

Reply:
I think "every push to master" creates quite a lot of overhead on the deployment side and many of the commits won't impact the demo at all. Every tagged release might be on the other extreme of the spectrum (too few updates). I'd suggest something in the middle (could be for a certain tag in a PR or we keep the `deploy-demo` branch and deploy for every push to it). What do you recommend @askainet? I guess the branch-based approach would be more similar to DC?